src/elements/app-data.html

```diff
@@ -84,14 +136,6 @@
           schedule: {
             type: Array,
             notify: true
-          },
-
-          _speakersRaw: {
```

Comment:
Why have you removed these properties if you use them in `_generateSchedule()` ?